build problem

运行
`npm run build`

```
> amphineko@0.0.1 build C:\Users\hasee\Desktop\amphineko-master
> node_modules/gulp/bin/gulp.js

node_modules' 不是内部或外部命令，也不是可运行的程序或批处理文件。
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amphineko@0.0.1 build: `node_modules/gulp/bin/gulp.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the amphineko@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\hasee\AppData\Roaming\npm-cache\_logs\2018-03-14T03_49_17_894Z-debug.log

```

尝试改成
`.\node_modules\.bin\gulp `
后OK

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
   },
   "homepage": "https://futa.moe/amphineko/",
   "scripts": {
-    "build": "node_modules/gulp/bin/gulp.js"
+    "build": "node ./node_modules/gulp/bin/gulp.js"
   },
   "devDependencies": {
     "gulp": "^3.9.1",

